releases/net472/KnownIssues/695400 - .NET 4.7.2 setup skips installing payload on Win10 machine when upgraded from Win7 with .NET 4.7.2 .md

```diff
@@ -0,0 +1,42 @@
+# .NET 4.7.2 setup fails to install files on OS when upgraded from Windows 7 SP1 or Windows Server 2008 R2 with .NET Framework 4.7.2 .
```

Comment:
The filename can be shortened:
` 695400 - setup skips installing payload on Windows 7 to Windows 10 migration.md`